README.md

```diff
@@ -106,12 +106,12 @@ interface "wasi:kv/data/readwrite" {
 
 interface "wasi:kv/data/atomic" {
   use { Error, key } from "wasi:kv/types"
-  
-  // atomically increments the value at key by one. 
+
+  // atomically increments the value at key by one.
```

Comment:
You probably want to make these `///` comments instead of `//` for proper generation of the `abi` file in the future.